fmatch/matcher.py

```diff
@@ -166,6 +166,43 @@ def filter_runs(self, pdata, data):
         ids_df = ndf.loc[df['jobConfig.jobIterations'] == iterations]
         return ids_df['uuid'].to_list()
 
+    def getResults(self, uuid: str, uuids: list, index_str: str, metrics: dict):
```

Comment:
Can we please rename ```metrics``` as ```metric```, this will make it easy reading and match with the code in orion.

Reply:
metrics here is a dict, while in each list item in metrics there is "metric" with a specific key value. Don't want that to be confusing

Reply:
changed it to "name" instead to be a little clearer and not get confused with forms of "metric(s)"